ballerina/tests/asb_admin_negative_test.bal

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org).
```

Reply:
Addressed in [efd46c2](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/efd46c235e1a6170116c6793391430efceff740d).